setup.py

```diff
@@ -45,13 +49,14 @@
     long_description=long_description,
     long_description_content_type="text/markdown",
     author="Jim Kitchen and Erik Welch",
+    author_email="erik.n.welch@gmail.com",
```

Comment:
@jim22k would you like to include your email here?

Reply:
sure